workers/email_sender/send_email_worker.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+from jinja2 import Environment, PackageLoader, select_autoescape
+
+from send_email_queue import connect_queue
+
+connection = connect_queue()
```

Comment:
Please move this in some kind of `main` function which must be called in 
`if __name__ == "__main__":` block